src/components/patients/index.tsx

```diff
@@ -2,19 +2,55 @@ import React from "react";
 import Header from "components/header";
 import PatientTable from "components/patients/patientTable";
 import SearchTool from "components/patients/searchTool";
+import { getPatients, getPatientsPerQuery } from "services/api";
+import { PatientBundle } from "types";
 import { Card, Elevation } from "@blueprintjs/core";
+import { requestNextPatients } from "services/api";
+
 import "./style.css";
 
+interface Props {
+  onSearch: Function;
+  searchItem: any;
+}
+
 const Patients = () => {
+  const [patientBundle, setPatientBundle] = React.useState({} as PatientBundle);
+
+  const getNextPatients = async () => {
+    const patBundle = (await requestNextPatients(
+      patientBundle
+    )) as PatientBundle;
```

Comment:
I don't think the `as` is needed, is it?

Reply:
The problem was that patBundle could be undefined if bundle.nextLink was undefined.
Modified like that :
```
    const patBundle = await requestNextPatients(patientBundle);
    if (patBundle) setPatientBundle(patBundle);
```